mathics/builtin/atomic/numbers.py

```diff
@@ -195,7 +195,7 @@ class Accuracy(Builtin):
      >> Accuracy[0.``2]
       = 2.
 
-    For 0.`, the accuracy is
+    For 0.`, the accuracy satisfies
```

Comment:
ok, but please add a colon after "satisfies" this what we do everywhere (and it happens to be the correct punctuation).